### PR TITLE
yast2_nis: add yast2 module nis client for SLES

### PR DESCRIPTION
**Pull request by Zaoliang**
* improve code of yast2_nis.pm
* add yast2_nis.pm into extratest of main.pm

### CODE REVIEW

**Comment by okurz** [comment]
as discussed in person:
* rename existing commit describing why the change of the module
* add the missing change for main.pm
* start commit message with capital letter after colon
* replace `wait_still_screen 1` by `wait_screen_change` around each call and test again

**Comment by Zaoliang** [comment]
okurz: changed now.

File: tests/console/yast2_nis.pm [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
+# Summary: create and delete nis client configuration and functionality
+# Maintainer: Zaoliang Luo <zluo@suse.de>
 
 use strict;
 use base "console_yasttest";
 use testapi;
 
 
-#test yast nis functionality, please see test results at http://e13.suse.de/tests/229
+# reference test: http://e13.suse.de/tests/1615
```
Comment (Soulofdestiny) [inline tests/console/yast2_nis.pm]: better place the link to the local verification run in the pr-description

File: tests/console/yast2_nis.pm [hunk 2]
```diff
@@ -58,18 +57,16 @@ sub run() {
...
+    wait_screen_change { send_key 'alt-o' };
     send_key 'alt-f';                                  # close the dialog...
     assert_screen 'nis_server_not_found';              # check error message for 'nis server not found'
     send_key 'alt-o';                                  # close it now even when config is not valid
+
 }
 1;
 
 # vim: set sw=4 et:
+
```
Comment (Soulofdestiny) [inline tests/console/yast2_nis.pm]: whats that empty line? ;)

Comment (Zaoliang) [inline tests/console/yast2_nis.pm]: empty line and line reference test: http://e13.suse.de/tests/1615
got removed now.

**Pull request merged**